README.md

```diff
@@ -28,23 +28,22 @@ Manage your Docker environment with ease.
 ## About
 
 Portainer is an open-source lightweight management UI which allows you to
-easily manage your a Docker host(s) or Docker swarm clusters.
+easily manage a Docker host(s) or Docker swarm clusters.
 
 It has never been so easy to manage Docker. Portainer provides a detailed
 overview of Docker and allows you to manage containers, images, networks and
 volumes.
 
 ## WARNING
 
-The Portainer add-on is really powerful and gives you virtually access to
+The Portainer add-on is really powerful and gives you access to virtually
 your whole system. While this add-on is created and maintained with care and
 with security in mind, in the wrong or inexperienced hands,
 it could damage your system.
 
 ## Installation
 
-The installation of this add-on is pretty straightforward and not different in
-comparison to installing any other Hass.io add-on.
+The installation of this add-on is pretty straightforward and no different compared to installing any other Hass.io add-on.
```

Comment:
```suggestion
The installation of this add-on is pretty straightforward and no different
compared to installing any other Hass.io add-on.
```

We try to limit lines to 80 chars.

Reply:
Line to long, not sure why you've marked this as resolved.